DOCS/client-api-changes.rst

```diff
@@ -31,7 +31,9 @@ API changes
 ===========
 
 ::
-
+ --- mpv 0.38.0 ---
+ 2.3    - revert the changes from API version 1.27, remove libmpv as the default VO and move it back
+          to the bottom of the auto-probing order.
```

Comment:
I don't think this is actually an API change is it?

Reply:
wm4 added an API change back in the days when he made that change. so i added one too for consistency. though i am fine with removing it, if wanted.

what it says:
```
 1.27   - make opengl-cb the default VO. This causes a subtle behavior change
          if the API user called mpv_opengl_cb_init_gl(), but does not set
          the "vo" option. Before, it would still have used another VO (like
          on the CLI, e.g. vo=gpu). Now it'll behave as if vo=opengl-cb was
          used.
```

Reply:
Nah that's fine. If there's already precedent, then we should just follow.